lib/Doctrine/ORM/Query/QueryException.php

```diff
@@ -204,13 +204,15 @@ public static function overwritingJoinConditionsNotYetSupported($assoc)
     }
 
     /**
+     * @param object $pathExpr
+     *
      * @return QueryException
      */
-    public static function associationPathInverseSideNotSupported()
+    public static function associationPathInverseSideNotSupported($pathExpr)
```

Comment:
[I was following this example.](https://github.com/Isinlor/doctrine2/blob/4f854ae7cca3f94df363321b5e9ab878c0580249/lib/Doctrine/ORM/Query/QueryException.php#L150) Are you fine with the inconsistency?